bmx-mode.el

```diff
@@ -6,7 +6,7 @@
 ;; URL: http://github.com/josteink/bmx-mode
 ;; Version: 0.1
 ;; Keywords: c convenience tools
-;; Package-Requires: ((cl-lib "0.5") (company "0.9.4") (dash "2.13.0") (s "1.12.0"))
+;; Package-Requires: ((emacs "25.1") (cl-lib "0.5") (company "0.9.4") (dash "2.13.0") (s "1.12.0"))
```

Comment:
bat-mode is also available via MELPA prior to Emacs 25.1. Wouldn’t requiring that give us better backwards compatibility? Or would that lead to other issues?

Reply:
I didn't find `bat-mode` in MELPA. Are you sure of it? If Yes, I could change my changes.

Reply:
Hm. It might have been removed in the big accidental wipe they had a while back, and not been restored due to originally being hosted on Emacs-wiki. 

In which case this is fine as is. No need to change anything!